frontend/src/container/NewWidget/LeftContainer/QuerySection/index.tsx

```diff
@@ -52,7 +52,7 @@ const QuerySection = ({
 				<Query
 					currentIndex={index}
 					selectedTime={selectedTime}
-					key={e.query + index}
+					key={`${e.query} ${e.query.length}`}
 					preQuery={e.query}
```

Comment:
This should be kept same or do you want to change something here as well?

Reply:
this is fine